tests/actors/rust/wasi-keyvalue-component/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+wit_bindgen::generate!({
+    world: "test-keyvalue",
+    exports: {
+        "wasmcloud:bus/guest": WasiKeyvalueComponent
```

Comment:
If this is meant to replicate what "users would do", why would we ever want users to directly implement `wasmcloud:bus/guest`?

Reply:
That's an option that users have -- or are you saying that `wasmcloud:bus/guest` was not written for people to be able to use, ever?

As far as I understand `wasmcloud:bus/guest` is the easiest way to build an actor that triggers a provider (we can't send invocations directly to providers without a bunch of hacks right now).